cmd/sonobuoy/app/gen.go

```diff
@@ -19,16 +19,19 @@ package app
 import (
 	"fmt"
 	"os"
+	"strings"
 
 	"github.com/pkg/errors"
 	"github.com/spf13/cobra"
 
-	"github.com/heptio/sonobuoy/cmd/sonobuoy/app/args"
 	ops "github.com/heptio/sonobuoy/cmd/sonobuoy/app/operations"
 	"github.com/heptio/sonobuoy/pkg/errlog"
 )
 
 var genopts ops.GenConfig
```

Comment:
I don't think this needs to be a global variable anymore. It only was before so we could use it from `init`.

Reply:
I must be missing something, it is still used by init.